style.css

```diff
@@ -334,3 +334,77 @@ button {
   line-height: 20px;
   color: #253858;
 }
+
+.contact-form {
+  background-image: url(src/Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contactContainer {
+  background-image: url(src/contact-shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+}
+
+.form-title {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
+
+.content {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #ebebff;
+  text-align: center;
+  margin: 10px;
+}
```

Comment:
- [ ] Kindly make the `margin: 10px;` bigger, to be like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155250455-412ac45a-ac45-43f2-81f2-fc6a8c42a553.png) | ![image](https://user-images.githubusercontent.com/35535130/155250491-f7580aa5-5c23-4f1b-a82b-906836ced9ba.png) |